### DISCUSSION TITLE
Why Are Git Variables Necessary for FDS Compilation?

### DISCUSSION THREAD

**Discussion by kjrstory** [firemodels/fds#13456] [Q&A]
I'm trying to install FDS, and it needs certain Git configuration variables during compilation, such as:
GITINFO_BASE, GITINFO, GITFOGNU
But if I only download the source code without using git clone, the .git folder is missing. 
Why are these Git variables necessary for compilation?

**Comment by drjfloyd** [comment]
The git variables are used during the build to identify the specific version of the FDS source that was compiled. This is in accordance with the FDS Software Quality Assurance Plan.  If you are going to be working the FDS source, using git will let you easily keep your source up to date.
